assets/js/modules/analytics-4/utils/data-mock.js

```diff
@@ -376,63 +386,143 @@ export function getAnalytics4MockResponse( args ) {
 		),
 	];
 
+	const metricAggregations = castArray( args.metricAggregations );
```

Comment:
Metric aggregations are currently unconditionally requested for all reports, so we should always include them here rather than making it an argument.

https://github.com/google/site-kit-wp/blob/4cb5777fae8d0d724629872917f13dbecf0424e4/includes/Modules/Analytics_4.php#L1083-L1090

Reply:
👍